src/main/java/org/candlepin/subscriptions/tally/facts/FactNormalizer.java

```diff
@@ -224,17 +225,36 @@ private void normalizeRhsmFacts(NormalizedFacts normalizedFacts, InventoryHostFa
                 normalizedFacts.getProducts().addAll(
                     roleToProductsMap.getOrDefault(hostFacts.getSyspurposeRole(), Collections.emptyList()));
             }
-            ServiceLevel effectiveSla = ServiceLevel.fromString(hostFacts.getSyspurposeSla());
-            normalizedFacts.setSla(effectiveSla);
-            if (hostFacts.getSyspurposeSla() != null && effectiveSla == ServiceLevel.UNSPECIFIED) {
-                log.warn(
-                    "Owner {} host {} has unsupported value for SLA: {}",
-                    hostFacts.getOrgId(),
-                    hostFacts.getSubscriptionManagerId(),
-                    hostFacts.getSyspurposeSla()
-                );
-            }
+
+            normalizedFacts.setSla(extractRhsmSla(hostFacts));
+            normalizedFacts.setUsage(extractRhsmUsage(hostFacts));
+        }
+    }
+
+    private Usage extractRhsmUsage(InventoryHostFacts hostFacts) {
+        Usage effectiveUsage = Usage.fromString(hostFacts.getSyspurposeUsage());
+        if (hostFacts.getSyspurposeUsage() != null && effectiveUsage == Usage.UNSPECIFIED) {
+            log.warn(
```

Comment:
Do we really want to warn on this?  There's not much we can do about it if the host has an peculiar usage set.